components/goffice/Makefile

```diff
@@ -10,20 +10,21 @@
 
```

Comment:
Please, move it to appropriate category (library) http://wiki.openindiana.org/oi/Categories+layout